cmd/celestia/rpc_test.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/celestiaorg/nmt/namespace"
+)
+
+func Test_parseNamespaceID(t *testing.T) {
+	type testCase struct {
+		name    string
+		param   string
+		want    namespace.ID
+		wantErr bool
+	}
+	testCases := []testCase{
+		{
+			name:  "8 byte hex encoded namespace ID gets right padded",
```

Comment:
[blocking] this claims to right pad but the implementation claims to left pad. I think left padding is desired behavior

Reply:
It actually left pads, but the unit tests are failing! Didn't catch until now

Reply:
fixed in [4a098ab](https://github.com/celestiaorg/celestia-node/pull/2325/commits/4a098abbd87ef34d5dbb7315a0d4cf52d55b379f)